test/priority_queue_test.dart

```diff
@@ -69,6 +69,11 @@ void testQueueBody<T>(
 
     expect(q.toList(), equals(elements));
     expect(q.toSet().toList(), equals(elements));
+    expect(q.toUnorderedList(), unorderedEquals(elements));
+    expect(q.unorderedElements, unorderedEquals(elements));
```

Comment:
Given the custom logic for concurrent modification it would be good to test that

Reply:
Done.